ros_ws/src/common/constants.h

```diff
@@ -66,4 +78,15 @@ const std::string TELEOP_MUX_NODE = "teleop_mux_node";
  */
 const int LOOP_RATE = 30;
 
+//Robot Parameters
+/**
+ * @brief The robots diameter from right wheel to left wheel
+ */
+const double ROBOT_BASE = 0.7;
+/**
+ * @brief The radius of the robot's wheels
+ */
+const double WHEEL_RAD = 0.19;
```

Comment:
These values should be moved to parameter configurations in the parameter server. We can keep it as-is for now, but please create an issue for it so we can do it in the near future.